### ISSUE TITLE
Support for alternative plain text formats, specially org-mode?

### ISSUE THREAD

**Issue by NightMachinery** [reorproject/reor#59]

**Comment by samlhuillier** [comment]
Yes this is definitely coming! Sorry, there have been lots of issues. I will get to this next week...Do you perhaps want to try working on a PR for this? I can help pointing you in the right direction

**Comment by NightMachinery** [comment]
@samlhuillier Thanks. I am currently rather busy, and I don't think the LLMs are quite there yet😅 So no PRs from me.

**Comment by samlhuillier** [comment]
Gotcha no worries! I'll work on this issue when I get a chance...

**Comment by nausher** [comment]
Just checking if this was fixed, or if there's a way to accomplish this via updating the configuration files.
